.github/workflows/e2e-functional.yml

```diff
@@ -7,7 +7,25 @@ on:
   pull_request:
     types:
         - labeled
-
+  schedule:
+    - cron: '0 0 * * *' #12 AM Daily
```

Comment:
Sure @devinbinnie , will keep it similar to nightly build

Reply:
So what I was suggesting was having the job be triggered as part of the nightly build, as a side task. So that after nightly was updated we would run tests on it.

Reply:
@devinbinnie I think I got it now.
just to confirm, if we ran it from [here](https://github.com/mattermost/desktop/blob/master/.github/workflows/nightly-builds.yaml#L44) after all nightly-main, nightly-rainforest is run, this should suffice right ?